seneca/smart_contract_user_libs/storage/tabular.py

```diff
@@ -23,7 +23,7 @@
 import datetime
 
 ex = None
-name_space = None
+name_space = 'test_tabular'
```

Comment:
I don' think we want to do this. The name_space should be explicitly set by the caller.

Reply:
corrected in latest version of this branch